### ISSUE TITLE
WCOW: build produces an unrunnable app image

### ISSUE THREAD

**Issue by jromero** [buildpacks/pack#800]
### Summary

When building a Windows container using the latest version of lifecycle 0.9.0 (which uses platform API 0.4).

---

##### Steps

1. On a WCOW setup run the task `make build-windows-apps` from https://github.com/buildpacks/samples/pull/85/files
2. `docker run -it --rm sample-batch-script-app:nanoserver-1809`

##### Current behavior

```
ERROR: failed to launch: cmd execute: exec: "cmd": executable file not found in %PATH%
```

##### Expected behavior

Application runs as expected.

---

##### pack info

```
Pack:
  Version:  0.0.0+git-5e9a21f
  OS/Arch:  windows/amd64

Default Lifecycle Version:  0.9.0

Config:
  experimental=true
```

##### docker info

```
Client:
 Debug Mode: false

Server:
 Containers: 1
  Running: 0
  Paused: 0
  Stopped: 1
 Images: 38
 Server Version: 19.03.12
 Storage Driver: windowsfilter
  Windows:
 Logging Driver: json-file
 Plugins:
  Volume: local
  Network: ics internal l2bridge l2tunnel nat null overlay private transparent
  Log: awslogs etwlogs fluentd gcplogs gelf json-file local logentries splunk syslog
 Swarm: inactive
 Default Isolation: hyperv
 Kernel Version: 10.0 18363 (18362.1.amd64fre.19h1_release.190318-1202)
 Operating System: Windows 10 Pro Version 1909 (OS Build 18363.1016)
 OSType: windows
 Architecture: x86_64
 CPUs: 2
 Total Memory: 7.999GiB
 Name: DESKTOP-D0G2N78
 ID: BKPH:BDOG:5IE3:DKQA:5BYF:D56J:QUHQ:AIUM:NRJR:YYL6:ZWKT:IYNW
 Docker Root Dir: C:\ProgramData\Docker
 Debug Mode: true
  File Descriptors: -1
  Goroutines: 93
  System Time: 2020-08-12T15:03:46.9912819-07:00
  EventsListeners: 1
 Registry: https://index.docker.io/v1/
 Labels:
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
 Product License: Community Engine
```

**Comment by jromero** [comment]
Related: https://github.com/buildpacks/rfcs/pull/114

TLDR; won't fix

**Comment by micahyoung** [comment]
The eventual spec change, which would necessitate changes in lifecycle are here: https://github.com/buildpacks/spec/pull/147
